src/proto/Aptos.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+syntax = "proto3";
+
+package TW.Aptos.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// TODO: typical balance transfer, add more fields needed to sign
+message TransferMessage {
+    int64 amount = 1;
+    int64 fee = 2;
+    string to = 3;
+}
+
+// TODO: Input data necessary to create a signed transaction.
+message SigningInput {
+    bytes private_key = 1;
```

Comment:
There are some common fields: `sequence_number`, `max_gas_amount`, `gas_unit_price` and expiration timestamp

Reply:
> There are some common fields: `sequence_number`, `max_gas_amount`, `gas_unit_price` and expiration timestamp

Noted, same remark as above.